docs/get-started/running-nodes.mdx

```diff
@@ -4,306 +4,218 @@ title: 'Running Etherlink Nodes'
+To build them yourself, see [Building the Etherlink kernel](#building-the-etherlink-kernel).
+
+For simplicity, these steps show how to run the node in observer mode.
+
+1. Get a built version of the Smart Rollup node binary, named `octez-smart-rollup-node`, in one of these ways:
+
+   - To set up a Smart Rollup node from Etherlink genesis, it's best to use recent Octez source code.
+   One commit that is known to work well is [`c5ed8f9`](https://gitlab.com/tezos/tezos/-/commit/c5ed8f9d5737cec03b29e5a7c3bf2169cc239ea8) from the `master` branch of the [Octez repository](https://gitlab.com/tezos/tezos).
+   For more information about building Octez binaries, see the Octez documentation: https://tezos.gitlab.io/index.html
+
+   - You can also use Octez binary builds.
+   Builds for Linux systems are available for the [x86_64](https://gitlab.com/tezos/tezos/-/jobs/6849203975/artifacts/browse/octez-binaries/x86_64/) architecture.
+
+1. Initialize the local context of the node, which is where it stores local data:
+
+   1. Set the environment variable `sr_observer_data_dir` to the directory where the node should store its local data.
+   1. Initialize the local context by running this command:
+
+      ```bash
+      octez-smart-rollup-node init observer config for sr1Ghq66tYK9y3r8CC1Tf8i8m5nxh8nTvZEf  \
+        with operators --data-dir $sr_observer_data_dir
+      ```
+
+      This command generates a minimal configuration file (`config.json`) in the local data folder:
+
+      ```json
+      { "smart-rollup-address": "sr1Ghq66tYK9y3r8CC1Tf8i8m5nxh8nTvZEf",
+        "smart-rollup-node-operator": {}, "fee-parameters": {},
+        "mode": "observer" }
+      ```
+
+1. Activate the `unsafe-pvm-patches` setting, which is required for Etherlink, and add the location of the preimages by adding these settings to the `config.json` file, as in this example:
+
+   ```json
+   { "smart-rollup-address": "sr1Ghq66tYK9y3r8CC1Tf8i8m5nxh8nTvZEf",
+     "smart-rollup-node-operator": {}, "fee-parameters": {},
+     "unsafe-pvm-patches": [
+       { "increase_max_nb_tick": "50_000_000_000_000" }
+     ],
+     "pre-images-endpoint": "https://snapshots.eu.tzinit.org/etherlink-mainnet/wasm_2_0_0",
+     "mode": "observer" }
+   ```
+
+   Similar to the Rust keyword, `unsafe` here means to be used with a good understanding of the feature.
+   Etherlink uses this setting with care and using it is safe in this context.
+
+1. Start the Smart Rollup node in observer mode by running this command and using the RPC endpoint of a layer 1 node that is running in archive mode:
+
+   ```bash
+   octez-smart-rollup-node --endpoint https://rpc.tzkt.io/mainnet run \
+     --data-dir $sr_observer_data_dir \
+     --apply-unsafe-patches
+   ```
+
+   As in this example, you can use a public layer 1 RPC node for testing purposes.
+   For production use, you must use a layer 1 node that is running in archive mode and provides access to its RPC endpoints.
```

Comment:
This is not strictly true. 
After the rollup node has catchup, there is no need to connect it to a archive l1 node. the archive node is needed in case the rollup node needs to catchup 

But it's reocmmanded to uses your own l1 node to not get disconnected by the l1 node, and maybe it's better to be connected to a archive node but it's not an obligation, specially for an observer node.

Reply:
How's this? 

>    As in this example, you can use a public layer 1 RPC node for initial setup, or you can connect it to a layer 1 node that you are running for a more stable connection. The Smart Rollup node needs a connection to an archive node while it catches up to the current state of layer 1; after that you can connect it to a rolling node.

Reply:
yes sounds good.